lib/system/web/src/resize_observer.rs

```diff
@@ -39,8 +39,8 @@ pub struct ResizeObserver {
 impl ResizeObserver {
     pub fn new(target: &JsValue, listener: Listener) -> Self {
         let target = target.clone();
-        let observer_id = resize_observe(&target, &listener);
-        Self { target, listener, observer_id }
+//        let observer_id = resize_observe(&target, &listener);
+        Self { target, listener, observer_id: 0 }
```

Comment:
I guess you are commenting it out because `resize_observe` is broken, not returning the index.

Check https://github.com/luna/basegl/pull/58/files#diff-d3e7c8016263f54e4e42b44c25a86dda

Reply:
Huh? _I have no memory of this place_ I think that comment was in the middle of some heavy refactoring. I try to just revert it